examples/vl3_basic/helm/vl3/templates/vl3-nse-ucnf.tpl

```diff
@@ -91,10 +93,22 @@ data:
         cnns/nsr.port: {{ .Values.cnns.nsr.port | quote }}
 {{- end }}
       cnns:
-        name: {{ .Values.nsm.serviceName | quote }}
-        address: "{{ .Values.cnns.nsr.addr }}:{{ .Values.cnns.nsr.port }}"
+        name: {{ .Values.cnns.nsr.name | quote }}
+        address: {{ .Values.cnns.nsr.addr | quote }}
+        connectivityDomain: {{ .Values.cnns.nsr.cd | quote}}
```

Comment:
I'm not sure why you're modifying these fields.  The connectivity domain is the same as the service name

Reply:
I'd expect modification of this for adding the ipamServer <address:port> here